Source/Dafny/AST/DafnyAst.cs

```diff
@@ -10528,13 +10550,15 @@ public class SeqSelectExpr : Expression {
     public readonly Expression Seq;
     public readonly Expression E0;
     public readonly Expression E1;
+    public readonly IToken CloseParens;
```

Comment:
```suggestion
    public readonly IToken CloseParen;
```

I find `CloseParens` implies "Close Parentheses" when it should be singular (and you used `CloseParen` elsewhere too)